<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CEIC爬虫</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="getDataGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>获取数据</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="type">
      <item>
       <widget class="QRadioButton" name="yearly">
        <property name="text">
         <string>年数据</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="monthly">
        <property name="text">
         <string>月数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="daily">
        <property name="text">
         <string>日数据</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="el">
      <property name="text">
       <string>止</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="endDate">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2014</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </widget>
    <widget class="QTextEdit" name="URL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>521</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="sl">
      <property name="text">
       <string>起</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="startDate">
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>1995</year>
        <month>9</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="GetData">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>获取数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>621</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>生成XLSX</string>
    </property>
    <widget class="QPushButton" name="Generate">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>32</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>数据名称</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Dataname"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="oneKey">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>一键获取并生成</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>621</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>输出信息</string>
    </property>
    <widget class="QPlainTextEdit" name="Output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
